actionview/test/template/text_test.rb

```diff
@@ -4,7 +4,7 @@
 
 class TextTest < ActiveSupport::TestCase
   test "formats always return :text" do
```

Comment:
@tenderlove Method `formats` was renamed to `format`. Can I update the test description here to `format`?

Reply:
Changed here #35406.